contracts/minitoken/solidity/proto/lib/Packet.proto

```diff
@@ -2,6 +2,9 @@ syntax = "proto3";
 
 option go_package = "pkg/ibc/app";
 
+import "solidity-protobuf-extensions.proto";
+option (.solidity.file_options) = { dirpath: "contracts/lib" };
```

Comment:
@3100 Unfortunately, the `dirpath` option will be discontinued.
https://github.com/datachainlab/solidity-protobuf/issues/32

You can run this example without using `dirpath`, by specifying `--sol_out=...:$(pwd)/contracts/lib`.

Reply:
@siburu Thank you. Fixed it as you suggested.